src/test/java/api/requests/scenarios/MoveRequestPolicyTests.java

```diff
@@ -0,0 +1,419 @@
+package api.requests.scenarios;
+
+import static java.util.Collections.singletonList;
+import static org.folio.circulation.domain.representations.RequestProperties.REQUEST_TYPE;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import org.folio.circulation.domain.MultipleRecords;
+import org.folio.circulation.domain.RequestType;
+import org.folio.circulation.domain.policy.Period;
+import org.folio.circulation.support.ClockManager;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.ISODateTimeFormat;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.MoveRequestBuilder;
+import api.support.builders.NoticeConfigurationBuilder;
+import api.support.builders.NoticePolicyBuilder;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Notes:<br>
+ *  MGD = Minimum guaranteed due date<br>
+ *  RD = Recall due date<br>
+ */
+public class MoveRequestPolicyTests extends APITests {
+  private static final String RECALL_TO_LOANEE = "Recall loanee";
+  private static Clock clock;
+
+  private NoticePolicyBuilder noticePolicy;
+
+  @BeforeClass
+  public static void setUpBeforeClass() {
+    clock = Clock.fixed(Instant.now(), ZoneOffset.UTC);
+  }
+
+  @Before
+  public void setUp() {
+    // reset the clock before each test (just in case)
+    ClockManager.getClockManager().setClock(clock);
+  }
+
+  @Before
+  public void setUpNoticePolicy() throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
```

Comment:
Why is the notice policy relevant for these tests?

Reply:
Ah, it probably is not. Let me go over it again.

Reply:
The test `moveRecallRequestWithoutExistingRecallsAndWithNoPolicyValuesChangesDueDateToSystemDate` is a scenario which must send a notice to loanee on recall.

https://issues.folio.org/browse/CIRC-316 scenario 2

Reply:
If sending notices are only needed for this specific test, then I think it could be clearer to only set up the policy for this test

Reply:
The `noticePolicy` is reused in a few tests.